src/System.Linq.Expressions/tests/Dynamic/DynamicObjectTests.cs

```diff
@@ -0,0 +1,791 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using Microsoft.CSharp.RuntimeBinder;
+using Xunit;
+
+namespace System.Dynamic.Tests
+{
+    public class DynamicObjectTests
+    {
+#pragma warning disable 649
```

Comment:
Can we fix this rather than suppress it?  Trying to remember warning code numbers... this is about unused fields?  If it's not easily fixed, please add a short comment about what this is for.